backend/src/datasource/post-datasource.js

```diff
@@ -0,0 +1,100 @@
+const {DataSource} = require('apollo-datasource');
+
+export class PostDatasource extends DataSource {
```

Comment:
⭐ For implementing a custom data source for apollo-server

Reply:
I started reviewing Abteilung6 yesterday. But I gave one comment and then had to interrupt my work. When I returned I noticed that Robert already reviewed the PR. I had nothing to add to Roberts review. You can find my comment here (https://github.com/Systems-Development-and-Frameworks/abteilung6/pull/5#discussion_r529623107)